sdk/cosmos/azure-cosmos-spark_3-0_2-12/pom.xml

```diff
@@ -279,6 +280,151 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version> <!-- {x-version-update;org.apache.maven.plugins:maven-shade-plugin;external_dependency} -->
```

Comment:
Nice - this dependency is used already and has been added to external dependencies :-)